javascripts/tileCollection.js

```diff
@@ -0,0 +1,57 @@
+function TileCollection(direction) {
+  // direction will be keypress values -- 37: left, 39: right, 38: up, or 40: down
+  this.direction = direction; // needed?
```

Comment:
Shouldn't be needed. It doesn't look like you reference `this.direction` anywhere.